app/controllers/reports_controller.rb

```
@@ -0,0 +1,70 @@
+  # the instructor's submitted response, the rubric items, the student
+  # calibration responses for the same reviewee, a per-item summary produced
+  # by CalibrationPerItemSummary, and the reviewee team's submitted content.
+  def calibration
+    instructor_map = ReviewResponseMap.find_by!(
+      id: params[:map_id],
+      reviewed_object_id: @assignment.id,
+      for_calibration: true
+    )
+
+    instructor_response = instructor_map.latest_submitted_response
+    return render_error('Submitted instructor calibration response not found', :unprocessable_entity) unless instructor_response
```

RuboCop (1.86.1)

[convention] Modifier form of `unless` makes the line too long.

(Style/IfUnlessModifier)